wordpress-mobile/WordPress-iOS
# Stats: Open `People -> Subscribers` from `Subscribers List` card

Partially addresses https://github.com/wordpress-mobile/WordPress-iOS/issues/23046
Continuation of Subscribers List Card PR (https://github.com/wordpress-mobile/WordPress-iOS/pull/23067)

Open `People -> Subscribers` from `Subscribers List` card with `Subscribers` tab selected. 

Other remaining tasks in other PRs:
- Waiting for new designs
- Remove Email Followers tab

## To test:

1. Enable Stats Traffic and Subscribers tab feature flag
2. Select site with more than 6 subscribers
3. Open Stats
4. Open Subscribers tab
5. Tap "View more" on Subscribers card
6. Confirm People view opens with "Subscribers" tab selected

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
